### DISCUSSION TITLE
Limit noisy users in my home feed

### DISCUSSION THREAD

**Discussion by NicolaeNMV** [mastodon/mastodon#21517] [Q&A] [answered]
Hi,

Some uses I follow post a lot of toots. Meaning I see multiple messages in the row from the same user but I don't see other users that toot less frequently.
Is there a way to limit how many toots I see from the same person ?

Thanks

**Comment by glw907** [comment, marked as answer]
Not that I've ever found. You can, however, hide boots from specific people. This isn't exactly what you asked, but it does help keep your feed quieter, especially if you've ended up following a "super-booster".

**Reply by NicolaeNMV** [reply]
Indeed, that's what's currently possible until there is a "smarter" home feed engine that could take this into account. Thanks for your comment @glw907 !
